app/models/manager_refresh/inventory_collection.rb

```diff
@@ -35,6 +36,7 @@ def initialize(model_class: nil, manager_ref: nil, association: nil, parent: nil
       @complete                         = complete.nil? ? true : complete
       @update_only                      = update_only.nil? ? false : update_only
       @builder_params                   = builder_params
+      @inventory_object_attributes      = inventory_object_attributes
```

Comment:
so now it will fallback to the method missing? Maybe we should have there a check, if inventory_object_attributes is defined, we will raise exception on unknown?

so then you can choose auto/manual